src/cascadia/TerminalApp/TerminalPage.xaml

```diff
@@ -58,6 +58,28 @@ the MIT License. See LICENSE in the project root for license information. -->
             x:Name="MultiLinePasteDialog"
             x:Uid="MultiLinePasteDialog"
             DefaultButton="Primary">
+            <StackPanel>
+                <TextBlock
+                    x:Name="MultiLineWarningText"
```

Comment:
We don't need to name infrastructural components unless we refer to them in code :smile: